app/controllers/admin/rebate_forms_controller.rb

```diff
@@ -65,6 +69,10 @@ def rebate_form_params
     params.require(:rebate_form).permit(attachments: [])
   end
 
+  def rebate_form_fields_params
+    params.fetch(:rebate_form).permit(:income, :dependants, :full_name, :dependants, :lived_here_before_july_2018, :full_name, :has_home_business, :email, :phone_number, :email_phone_can_be_used)
```

Comment:
Dependants is here twice.

We'll need to update the year every time. Can we do something smarter and resilient here?

Reply:
Did someone request all these extra fields? I thought it was just the 4 fields as written on the issue

Reply:
@Br3nda you mean like params.require(:rebate_form).permit! ? that way I dont need to specify the attributes. What is general practise.